File: test/export/pas/03-estructuras.pas
Program sin_titulo;
Var
	c: Real;
Begin
	c := 0;
	While c<=10 Do Begin
		c := c+1;
		WriteLn(c);
	End;
	c := 0;
	Repeat
		c := c+1;
		WriteLn(c);
	Until c=10;
	c := 0;
	Repeat
		c := c+1;
		WriteLn(c);
	Until c>=10;
	If c=10 Then Begin
		WriteLn('Si');
	End;
	If c=10 Then Begin
		WriteLn('Si');
	End
	Else Begin
		WriteLn('No');
	End;
End.

File: test/export/java/03-estructuras.java
import java.io.*;
public class sin_titulo {
	public static void main(String args[]) {
		double c;
		c = 0;
		while (c<=10) {
			c = c+1;
			System.out.println(c);
		}
		c = 0;
		do {
			c = c+1;
			System.out.println(c);
		} while (c!=10);
		c = 0;
		do {
			c = c+1;
			System.out.println(c);
		} while (c<10);
		if (c==10) {
			System.out.println("Si");
		}
		if (c==10) {
			System.out.println("Si");
		} else {
			System.out.println("No");
		}
	}
}
